brut.apktool/apktool-lib/src/main/java/brut/androlib/res/data/ResResSpec.java

```diff
@@ -28,11 +29,13 @@
 public class ResResSpec {
     private final ResID mId;
     private final String mName;
+    private final int mFlags;
     private final ResPackage mPackage;
     private final ResTypeSpec mType;
     private final Map<ResConfigFlags, ResResource> mResources = new LinkedHashMap<>();
 
-    public ResResSpec(ResID id, String name, ResPackage pkg, ResTypeSpec type) {
+    public ResResSpec(ResID id, String name, int mFlags, ResPackage pkg, ResTypeSpec type) {
```

Comment:
Normally we use `mXXX` to represent a member variable of the class. A dated dumb rule I think, but in the scope of the constructor - it should probably be just `flags`

Reply:
The `mFlags` is correct for the class variable though, so it becomes `this.mFlags = flags`

Reply:
Oh, yes, I know about it, I just copied it from the field and forgot to rename it.